docs/resource/process.md

```diff
@@ -32,10 +32,12 @@
 | [`process.command_line`](../attributes-registry/process.md) | string | The full command used to launch the process as a single string representing the full command. On Windows, can be set to the result of `GetCommandLineW`. Do not set this if you have to assemble it just for monitoring; use `process.command_args` instead. | `C:\cmd\otecol --config="my directory\config.yaml"` | Conditionally Required: See alternative attributes below. |
 | [`process.executable.name`](../attributes-registry/process.md) | string | The name of the process executable. On Linux based systems, can be set to the `Name` in `proc/[pid]/status`. On Windows, can be set to the base name of `GetProcessImageFileNameW`. | `otelcol` | Conditionally Required: See alternative attributes below. |
 | [`process.executable.path`](../attributes-registry/process.md) | string | The full path to the process executable. On Linux based systems, can be set to the target of `proc/[pid]/exe`. On Windows, can be set to the result of `GetProcessImageFileNameW`. | `/usr/bin/cmd/otelcol` | Conditionally Required: See alternative attributes below. |
-| [`process.owner`](../attributes-registry/process.md) | string | The username of the user that owns the process. | `root` | Recommended |
-| [`process.parent_pid`](../attributes-registry/process.md) | int | Parent Process identifier (PID). | `111` | Recommended |
+| [`process.owner`](../attributes-registry/process.md) | string | The username of the user that owns the process. [1] | `root` | Recommended |
+| [`process.parent_pid`](../attributes-registry/process.md) | int | Parent Process identifier (PPID). | `111` | Recommended |
```

Comment:
Why only these two got updated, and other new attributes not added to this file? Was this done manually?

Also, changes to the existing attributes are probably better to be done in a separate PR since it's not aligned with this PR's goal

Reply:
> I'll take out the note on process.owner, I agree that's probably worth its own discussion and PR, but I'd like to leave in PPID. PPID is the correct abbreviation for this, and I think it was probably a mistake that it was PID before.

Sure, we need to fix this. But it's still outside of this PR's goal. So can be another easy-to-merge bugfix PR.

Reply:
I've taken out both those changes